core/variant/variant_utility.cpp

```diff
@@ -1625,6 +1623,50 @@ static _FORCE_INLINE_ Variant::Type get_ret_type_helper(void (*p_func)(P...)) {
 	};                                                                                                           \
 	register_utility_function<Func_##m_func>(#m_func, m_args)
 
+#define FUNCBINDVARARGV_CNAME(m_func, m_func_cname, m_args, m_category)                                          \
+	class Func_##m_func {                                                                                        \
+	public:                                                                                                      \
+		static void call(Variant *r_ret, const Variant **p_args, int p_argcount, Callable::CallError &r_error) { \
+			r_error.error = Callable::CallError::CALL_OK;                                                        \
+			VariantUtilityFunctions::m_func_cname(p_args, p_argcount, r_error);                                  \
+		}                                                                                                        \
+		static void validated_call(Variant *r_ret, const Variant **p_args, int p_argcount) {                     \
+			Callable::CallError c;                                                                               \
+			VariantUtilityFunctions::m_func_cname(p_args, p_argcount, c);                                        \
+		}                                                                                                        \
+		static void ptrcall(void *ret, const void **p_args, int p_argcount) {                                    \
+			Vector<Variant> args;                                                                                \
```

Comment:
push_back in a loop :s . ould be better to resize + direct assignment, to ensure only one allocation.

Reply:
Agreed, but this is copied as is from `FUNCBINDVARARGV` (the only change is the c++ function name), so any improvements to efficiency here is out of scope.

Actually looking at it, it might be better to call this new macro from `FUNCBINDVARARGV`, to reduce the code duplication and make it clearer, so I'll change this.